splinterd/src/config/default.rs

```diff
@@ -14,8 +14,9 @@
 
 use crate::config::{ConfigError, ConfigSource, PartialConfig, PartialConfigBuilder};
 
-const DEFAULT_CERT_DIR: &str = "/etc/splinter/certs/";
-const DEFAULT_STATE_DIR: &str = "/var/lib/splinter/";
+pub(in crate::config) const CONFIG_DIR: &str = "/etc/splinter";
```

Comment:
These should remain private to this module.

Reply:
Fixed